### PR TITLE
Correction to postgre sql setup with node

### PR DESCRIPTION
**Pull request by jonathanglassman**
## What

Driven by story: https://www.pivotaltracker.com/story/show/153602295

Changes to PostgreSQL setup with Node.js:
- Corrected "mypostgres" to "SERVICE-NAME" in both code and paragraph below to ensure consistency
- Changed content to be more direct to user
- Added in two links to cf env resources
- Removed superfluous backticks in code at end of section
## How to review

Describe the steps required to test the changes.

1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.
1. Manually check that any removed or renamed anchor tags are not in use ([linkchecker](https://github.com/linkcheck/linkchecker) doesn't do this).
1. Check that content addresses story requirements

## Who can review

Anyone except Jon Glassman

### CODE REVIEW

File: source/documentation/deploying_apps/deploying_node_js.md [hunk 1]
```diff
@@ -89,22 +89,25 @@ Then in your app, you can easily get configuration information for backing servi
...
         client.ssl = true;
         client.connect();
 
-Note that in the above you should replace "my-postgres" with the exact name of the PostgreSQL service you created. The ``getServiceURL`` function returns a connection string which includes the username and password required to connect to the database.
+Replace "SERVICE NAME" in the above code with the exact name of the PostgreSQL service you created. The ``getServiceURL`` function returns a connection string which includes the username and password required to connect to the database. 
+
+Further information can be found on:
+
+- the Cloud Foundry [community `cf env` page](https://github.com/cloudfoundry-community/node-cfenv/blob/master/README.md) [external link]
+- the Cloud Foundry [documentation on environment variables](https://docs.run.pivotal.io/devguide/deploy-apps/environment-variable.html) [external link]
```
Comment (camelpunch) [inline source/documentation/deploying_apps/deploying_node_js.md]: We should probably link to https://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html instead. There are sometimes subtle differences between the PWS documentation (run.pivotal.io) and general CF docs.

File: source/documentation/deploying_apps/deploying_node_js.md [hunk 2]
```diff
@@ -89,22 +89,25 @@ Then in your app, you can easily get configuration information for backing servi
...
         var pg = require('pg');
         var appEnv = cfenv.getAppEnv();
-        var connectionString = appEnv.getServiceURL(/.*/);
+        var connectionString = appEnv.getServiceURL("SERVICE NAME");
         var client = new pg.Client(connectionString);
         client.ssl = true;
         client.connect();
 
-Note that in the above you should replace "my-postgres" with the exact name of the PostgreSQL service you created. The ``getServiceURL`` function returns a connection string which includes the username and password required to connect to the database.
+Replace "SERVICE NAME" in the above code with the exact name of the PostgreSQL service you created. The ``getServiceURL`` function returns a connection string which includes the username and password required to connect to the database.
```
Comment (camelpunch) [inline source/documentation/deploying_apps/deploying_node_js.md]: Very minor: there's a trailing space at the end of this paragraph.

**Comment by camelpunch** [comment]
Looks like GitHub has collapsed my main comment above as being 'outdated'. :(

File: source/documentation/deploying_apps/deploying_node_js.md (hunk 1 above)
Comment (jonathanglassman) [reply source/documentation/deploying_apps/deploying_node_js.md]: @camelpunch is the new link to replace both of the old links or just one of them? If so, which one?

Comment (jonathanglassman) [reply source/documentation/deploying_apps/deploying_node_js.md]: @camelpunch nvm, Alex just explained it is only for the second link

**Pull request merged**